### DISCUSSION TITLE
How to get html lsp completion to work?

### DISCUSSION THREAD

**Discussion by Multirious** [helix-editor/helix#5937] [Q&A]
I have installed [this](https://github.com/hrsh7th/vscode-langservers-extracted) from following [wiki](https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers#html) but the completion is not working and I'm not really sure what to do.

**Comment by danillos** [comment]
This feature is in progress on this PR https://github.com/helix-editor/helix/pull/5864. You can build from this PR and test.
